Intermittent phantom test failures

I see that often phantom tests timeout on spinner test. Can we increase the timeout values?

JS 1.9.8 (Linux 0.0.0): Executed 99 of 107 SUCCESS (0 secs / 0.255 secs)
JS 1.9.8 (Linux 0.0.0): Executed 100 of 107 SUCCESS (0 secs / 0.256 secs)
PhantomJS 1.9.8 (Linux 0.0.0) elgg/spinner start() adds the body class after 20ms FAILED
	Expected 0 to be 1.
	Error: Expected 0 to be 1.
	    at /home/travis/build/Elgg/Elgg/js/tests/ElggSpinnerTest.js:27
PhantomJS 1.9.8 (Linux 0.0.0): Executed 101 of 107 (1 FAILED) (0 secs / 0.282 secs)
JS 1.9.8 (Linux 0.0.0): Executed 102 of 107 (1 FAILED) (0 secs / 0.282 secs)
PhantomJS 1.9.8 (Linux 0.0.0) elgg/spinner start(text) shows escaped text below the spinner FAILED
	Expected 0 to be 1.
	Error: Expected 0 to be 1.
	    at /home/travis/build/Elgg/Elgg/js/tests/ElggSpinnerTest.js:42
PhantomJS 1.9.8 (Linux 0.0.0): Executed 103 of 107 (2 FAILED) (0 secs / 0.308 secs)
JS 1.9.8 (Linux 0.0.0): Executed 104 of 107 (2 FAILED) (0 secs / 0.333 secs)
JS 1.9.8 (Linux 0.0.0): Executed 105 of 107 (2 FAILED) (0 secs / 0.369 secs)
JS 1.9.8 (Linux 0.0.0): Executed 106 of 107 (2 FAILED) (0 secs / 0.404 secs)
JS 1.9.8 (Linux 0.0.0): Executed 107 of 107 (2 FAILED) (0 secs / 0.43 secs)
JS 1.9.8 (Linux 0.0.0): Executed 107 of 107 (2 FAILED) (0.433 secs / 0.43 secs)
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0
The command "npm test" exited with 1.
Done. Your build exited with 1.

--- a/js/tests/ElggSpinnerTest.js
+++ b/js/tests/ElggSpinnerTest.js
@@ -26,7 +26,7 @@ define(function(require) {
 			setTimeout(function() {
 				expect($(visible_selector).length).toBe(1);
 				done();
-			}, 25);
+			}, 40);
 		});
 
 		it("stop() removes the body class", function(done) {
@@ -37,7 +37,7 @@ define(function(require) {
 				spinner.stop();
 				expect($(visible_selector).length).toBe(0);
 				done();
-			}, 25);
+			}, 40);
 		});
 	});
 });

